lib/index.ts

```diff
@@ -158,15 +153,16 @@ class PWMetrics {
     }
   }
 
-  async launchChrome(): Promise<ChromeLauncher> {
+  async launchChrome(): Promise<LaunchedChrome|Error> {
     try {
-      this.launcher = new ChromeLauncher();
-      await this.launcher.isDebuggerReady();
-      return this.launcher;
-    } catch(e) {
       console.log(messages.getMessage('LAUNCHING_CHROME'));
-      await this.launcher.run();
-      return this.launcher;
+      return this.launcher = await launch({
+        port: this.flags.port,
```

Comment:
Tried it locally but I get this:
```
Error: connect ECONNREFUSED 127.0.0.1:9222
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1136:14)
```

Chrome opens a window with a tab but nothing happens.